### ISSUE TITLE
erro

### ISSUE THREAD

**Issue by mrym41** [eldraco/domain_analyzer#24]
i am using ubuntu os  and this command is not working domain_analyzer.py -d .gov -k 10 -b

**Comment by verovaleros** [comment]
Can you maybe share a screenshot? Which Python version are you using? What is the error exactly? Thank you

**Comment by mrym41** [comment]
Am using python 3

Thank you

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by mrym41** [comment]
am using pyhton 3 
![Uploading WhatsApp Image 2023-01-17 at 9.03.39 AM.jpeg…]()

**Comment by mrym41** [comment]
![WhatsApp Image 2023-01-17 at 9 03 39 AM(1)](https://user-images.githubusercontent.com/115137405/212856151-56d77cfe-f905-4b64-8f9c-52fa031f2d3a.jpeg)

**Comment by verovaleros** [comment]
Thank you for your screenshot. It seems your system is trying to find the `domain_analyzer.py` in your system-installed applications and not executing the Python in the directory.

Try running it like this:
- `./domain_analyzer.py -d .gov -k 10 -b`
- `python3 domain_analyzer.py -d .gov -k 10 -b`

**Comment by mrym41** [comment]
Still not working

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by mrym41** [comment]
![WhatsApp Image 2023-01-17 at 9 16 34 AM](https://user-images.githubusercontent.com/115137405/212858034-67221581-98fd-43ae-bea1-72fe1beac7f5.jpeg)

**Comment by verovaleros** [comment]
As the Google API changed, we are still working on migrating it to make it work.
I suggest dropping the `-k 10` option. 

Try something like this to check if the tool works:
- `python3 domain_analyzer.py -d example.com -b`

We are trying to find alternatives to the Google search to provide this functionality again, but it will take a bit of time. Thanks for your understanding.

**Comment by mrym41** [comment]
![ex](https://user-images.githubusercontent.com/115137405/212861615-e67d2fc5-1e62-4565-bc04-099996c327a9.jpeg)

**Comment by mrym41** [comment]
its show like this ?

**Comment by verovaleros** [comment]
Yes, that's correct. If you see, `domain_analyzer` is running but the error tells you that you don't have `nmap` installed.

**Comment by mrym41** [comment]
so the problem is google API ?
